Console/Command/RemoveUnusedMediaCommand.php

```diff
@@ -188,7 +190,7 @@ public function execute(InputInterface $input, OutputInterface $output): int
             }
 
             $filePathWithoutCacheDir = preg_replace('#/cache_*/[a-z0-9]+(/[a-z0-9]/[a-z0-9]/.+?)#i', '$1', $filePath);
-            if (in_array($filePathWithoutCacheDir, $imagesToKeep, true)) {
+            if (key_exists($filePathWithoutCacheDir, $imagesToKeep)) {
```

Comment:
```suggestion
            if (array_key_exists($filePathWithoutCacheDir, $imagesToKeep)) {
```

IMHO, it is clearer to not use the alias, but the actual function.